src/ot_orchestration/utils/utils.py

```diff
@@ -10,6 +11,16 @@
 from google.cloud.storage import Client
 
 
+def bucket_name(gs_url: str) -> str:
```

Comment:
This could be used in `GCSPath` (in case you require the full path object, I you might want to add a method there.

Reply:
Indeed, actually we can use the `split` method you created. I've updated my code.